backend/RestAPI/Controllers/VideosController.cs

```diff
@@ -192,25 +192,33 @@ public async Task<IActionResult> DeleteVideo([FromRoute] Guid id)
                 return NotFound();
             }
 
-            if (id == Guid.Empty) return BadRequest();
-
-            var video = await _videosRepository.GetVideoById(id);
-            if (video == null)
+            foreach (Guid id in ids)
             {
-                return NotFound();
-            }
+                if (id == Guid.Empty) return BadRequest();
 
-            try
-            {
-                await _videoService.DeleteVideo(video, user.Id);
-            }
-            catch (FileNotFoundException ex)
-            {
-                return BadRequest(ex.Message);
-            }
-            catch
-            {
-                return StatusCode(500);
+                var video = await _videosRepository.GetVideoById(id);
```

Comment:
Same as for Aurimas, should we be calling a repository in controller

Reply:
It is totally fine to do that. @povilassta You can merge this PR